vaadin-markdown-flow-parent/vaadin-markdown-flow-integration-tests/src/test/java/com/vaadin/flow/component/markdown/tests/MarkdownIT.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.markdown.tests;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.vaadin.flow.component.markdown.testbench.MarkdownElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.AbstractComponentIT;
+
+@TestPath("markdown")
+public class MarkdownIT extends AbstractComponentIT {
+
+    private MarkdownElement markdownElement;
+
+    @Before
+    public void init() {
+        open();
+        markdownElement = $(MarkdownElement.class).waitForFirst();
+    }
+
+    @Test
+    public void sizeMatches() {
```

Comment:
Is that something specific to the Flow component? Seems related to the `HasSize` API, which probably doesn't need testing.

Reply:
Removed the unnecessary test